content/en/docs/gke/private-clusters.md

```diff
@@ -1,307 +1,247 @@
 +++
 title = "Securing Your Clusters"
-description = "How to secure Kubeflow clusters using VPC service controls and private GKE"
+description = "How to secure Kubeflow clusters using private GKE"
 weight = 70
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
-
-{{% alert title="Alpha" color="warning" %}}
-This feature is currently in **alpha** release status with limited support. The
-Kubeflow team is interested in any feedback you may have, in particular with 
-regards to usability of the feature. Note the following issues already reported:
-
-* [Documentation on how to use Kubeflow with private GKE and VPC service controls](https://github.com/kubeflow/website/issues/1705)
-* [Replicating Docker images to private Container Registry](https://github.com/kubeflow/kubeflow/issues/3210)
-* [Installing Istio for Kubeflow on private GKE](https://github.com/kubeflow/kubeflow/issues/3650)
-* [Profile-controller crashes on GKE private cluster](https://github.com/kubeflow/kubeflow/issues/4661)
-* [kfctl should work with private GKE without public endpoint](https://github.com/kubeflow/kfctl/issues/158)
-{{% /alert %}}
-
-This guide describes how to secure Kubeflow using [VPC Service Controls](https://cloud.google.com/vpc-service-controls/docs/) and private GKE.
-
-Together these two features signficantly increase security
-and mitigate the risk of data exfiltration.
-
-  * VPC Service Controls allow you to define a perimeter around
-    Google Cloud Platform (GCP) services.
-    
-    Kubeflow uses VPC Service Controls to prevent applications
-    running on GKE from writing data to GCP resources outside
-    the perimeter.
 
-  * Private GKE removes public IP addresses from GKE nodes making
-    them inaccessible from the public internet.
+These instructions explain how to deploy Kubeflow using private GKE.
 
-    Kubeflow uses IAP to make Kubeflow web apps accessible
-    from your browser.
+1. Follow the blueprint instructions to setup a management cluster
```

Comment:
Line 10: The steps below `1. Follow the blueprint instructions to setup a management cluster` - should be be indented to the right?